### PR TITLE
(DOCS-8996) Add limits page for Workflow Automation

### PR DESCRIPTION
**Pull request by neko-dd**
### What does this PR do? What is the motivation?

Add a page for workflow automation rate limits and throttles

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/service_management/workflows/limits.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+* A workflow can contain up to 100 steps. If you need more than 100 steps in a workflow, you can use the **Trigger workflow** step to [call a child workflow][2]. Use output parameters to pass the output of a child workflow back to your main workflow.
+* A workflow can run for up to 24 hours. Workflows terminate when they attempt to run for longer than 24 hours.
+* A workflow can start up to 30 steps per minute. If you exceed this rate, steps are throttled and start at a rate of 30 per minute.
+* The sum of all step outputs for a workflow can be up to 150 MB.
+* A workflow's output size can be up to 500 KB.
+
+## End user limits
+
+Each end user can execute up to 100 actions per minute across App Builder and Workflow Automation.
```
Comment (hestonhoffman) [inline content/en/service_management/workflows/limits.md:53]:
```suggestion
Each end user can execute up to 100 actions per minute across App Builder and Workflow Automation.
```
So an org can execute 200 workflow runs per minute, but a user can only execute 100 actions (and each workflow/app has at least one action - many will probably hit the max). Perhaps we need another paragraph here about how buffering works per user - I imaging it's the same as above.

File: content/en/service_management/workflows/trigger.md [hunk 2]
```diff
@@ -202,6 +202,8 @@ If the child workflow has [input parameters][5], these parameters appear as requ
 
 {{< img src="service_management/workflows/trigger-workflow-step.png" alt="The service_name input parameter is required in the child workflow" style="width:100%;" >}}
 
+You can also pass the result of a child workflow back to the parent workflow by defining **Output parameters** in the child workflow. The **Trigger Workflow** step in the parent workflow retrieves the output parameters from the child workflow.
```
Comment (hestonhoffman) [inline content/en/service_management/workflows/trigger.md:205]:
```suggestion
You can also pass the result of a child workflow back to the parent workflow by defining **Output parameters** in the child workflow. Use the `WorkflowOutputs` context variable in the parent workflow to retrieve the output parameters from the child workflow. For example, given a child workflow named `Example_workflow` with an output parameter named `exampleList`, use `Steps.Example_workflow.workflowOutputs.exampleList` to access the result from the child workflow.
```

**Review by hestonhoffman — APPROVED** [review]
Looks good. I left some small suggestions

**Review by ambertunnell — APPROVED** [review]
Looks good!

**Pull request merged**